server/client_manager.py

```diff
@@ -106,52 +113,57 @@ def change_area(self, area):
             self.send_command('HP', 1, self.area.hp_def)
             self.send_command('HP', 2, self.area.hp_pro)
             self.send_command('BN', self.area.background)
-            self.send_command('LE', *self.area.get_evidence_list())
+            self.send_command('LE', *self.area.get_evidence_list(self))
 
         def send_area_list(self):
             msg = '=== Areas ==='
+            lock = {True: '[LOCKED]', False: ''}
             for i, area in enumerate(self.server.area_manager.areas):
-                msg += '\r\nArea {}: {} (users: {})'.format(i, area.name, len(area.clients))
-                if self.area == area:
-                    msg += ' [*]'
-                msg += '\r\n[{}]'.format(area.status)
-                if area.is_locked:
-                    msg += '[LOCKED]'
-            self.send_host_message(msg)
-
-        def send_limited_area_list(self):
-            msg = '=== Areas ==='
-            for i, area in enumerate(self.server.area_manager.areas):
-                msg += '\r\nArea {}: {}'.format(i, area.name)
+                owner = 'FREE'
+                if area.owned:
+                    for client in area.clients:
+                        if client.is_cm:
```

Comment:
I'm pretty sure there's a more Pythonic way to search for clients meeting a certain predicate